### PR TITLE
feat: create separate templates for each component

### PR DESCRIPTION
**Pull request by GusBaamonde**
templates for each component

### CODE REVIEW

**Comment by andresmoschini** [comment]
What do you think about adding links in the index page? 

Right now I am only seeing some colors:

image

File: assets/templates/messages-header.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  </div>
+</div><!-- error -->
+<!-- amarillo -->
+<div class="messages-container warning">
+  <div class="wrapper">
+    <p>Thanks! We’ll contact you ASAP and if everything is fine, you’ll be able to send your Campaign.</p>
+  </div>
+</div><!-- warning-->
+<!-- verde -->
+<div class="messages-container message">
```
Comment (fcoronelmakingsense) [inline assets/templates/messages-header.html]: can add css class 'info' with the same styles that 'message'. We need for the webapp. Thanks!

**Comment by andresmoschini** [comment]
image

Please, try to rebase in place of backward merge.

**Pull request merged**